### ISSUE TITLE
Annoying deprecation warning

### ISSUE THREAD

**Issue by fvictorio** [browserify/resolve#230]
`resolve@1.18.0` shows a deprecation warning when it's required:

```
> require('resolve');
`resolve/lib/core` is deprecated; please use `is-core-module` directly
```

This seems to me like it's being shown by mistake, since it's not the whole `resolve` that is deprecated, so why would it be shown with a plain require? Am I missing something here? Thanks!

**Comment by ljharb** [comment]
Hmm, you're right, that shouldn't happen just on requiring the entry point. I'll fix that soon.

**Comment by ljharb** [comment]
Fixed in v1.18.1.

**Comment by fvictorio** [comment]
Thank you!
